### PR TITLE
Generate superflore-datetime.inc

### PR DESCRIPTION
**Pull request by andre-rosa**
- As per spec r35

### CODE REVIEW

**Comment by andre-rosa** [comment]
Fixes #202 .

**Comment by andre-rosa** [comment]
@allenh1 << Ptal

**Comment by andre-rosa** [comment]
@allenh1 @tfoote << Ptal :)

File: superflore/generators/bitbake/yocto_recipe.py [hunk 1]
```diff
@@ -499,24 +500,42 @@ def get_recipe_text(self, distributor):
...
+        try:
+            make_dir(datetime_dir)
+            with open(datetime_path, 'w') as datetime_file:
+                datetime_file.write('# {}/{}\n'.format(
+                    dist, datetime_file_name))
+                datetime_file.write('# Generated by superflore -- DO NOT EDIT')
+                datetime_file.write(
+                    ' (except ROS_DISTRO_METADATA_VERSION_REVISION)\n#\n')
+                datetime_file.write(
+                    '# Copyright 2019 Open Source Robotics Foundation\n\n')
```
Comment (tfoote) [inline superflore/generators/bitbake/yocto_recipe.py]: Can we make this use the current year not hard code 2019 for future maintenance avoidance.

Comment (allenh1) [reply superflore/generators/bitbake/yocto_recipe.py]: @andre-rosa Please implement this and I'll merge it in

Comment (andre-rosa) [reply superflore/generators/bitbake/yocto_recipe.py]: Thanks @tfoote, makes sense.

**Comment by andre-rosa** [comment]
Ptal :)

**Pull request merged**